cmd/tailscaled/tailscaled.go

```diff
@@ -238,8 +239,19 @@ func ipnServerOpts() (o ipnserver.Options) {
 			o.VarRoot = dir
 		}
 	}
+	if strings.HasPrefix(statePathOrDefault(), "ephemeral:") {
+		// Register as an ephemeral node.
+		o.LoginFlags |= controlclient.LoginEphemeral
```

Comment:
no need for the `|` .... we made this `o` and there's no assignment to `LoginFlags` before this.

Reply:
Done